constellations/orion/src/tests_integration/user_class_integration_test.go

```diff
@@ -0,0 +1,290 @@
+package tests_integration
+
+import (
+	"encoding/json"
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/tests_integration/utils"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+// Test: Create UserClasses and Get by GetClassesByUserId()
+func Test_CreateUserClasses(t *testing.T) {
+	createAccountUser(t)
+	createClasses(t)
+
+	createAllUserClasses(t)
+
+	// Call Get All!
+	recorder7 := utils.SendHttpRequest(t, http.MethodGet, "/api/userclass/user/1", nil)
+
+	// Validate results
+	assert.EqualValues(t, http.StatusOK, recorder7.Code)
+
+	var userClass []domains.UserClass
+	if err := json.Unmarshal(recorder7.Body.Bytes(), &userClass); err != nil {
+		t.Errorf("unexpected error: %v\n", err)
+	}
+	assertUserClass(t, 1, userClass[0])
+	assertUserClass(t, 2, userClass[1])
+
+	assert.EqualValues(t, 2, len(userClass))
+
+	utils.ResetTable(t, domains.TABLE_USERCLASS)
+	utils.ResetTable(t, domains.TABLE_USERS)
+	utils.ResetTable(t, domains.TABLE_CLASSES)
+	utils.ResetTable(t, domains.TABLE_PROGRAMS)
+	utils.ResetTable(t, domains.TABLE_SEMESTERS)
+	utils.ResetTable(t, domains.TABLE_LOCATIONS)
+	utils.ResetTable(t, domains.TABLE_ACCOUNTS)
```

Comment:
move this to its own function at the bottom of this file as well.
Also reorder then by how they're related
i.e. first remove user_class -> user -> account. Then class -> programs -> etc.